tests/e2e/genesis.go

```diff
@@ -0,0 +1,110 @@
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	tmtypes "github.com/tendermint/tendermint/types"
```

Comment:
group imports. tendermint should go above cosmos-sdk

Reply:
I have grouped, please check again.